src/html/components/footer.html

```diff
@@ -0,0 +1,9 @@
+<footer>
+  <div class="logo-with-title">
+    <h2>An Exploration By</h2>
+    <img src="images/viget-logo.svg" alt="Viget Logo" class="logo">
```

Comment:
In addition to Ben's suggestion, you could remove the alt text here, since we've already identified to screen reader users that this was "by Viget."